frontend/src/features/mapSideBar/tabs/Section.tsx

```diff
@@ -18,7 +18,7 @@ export const Section: React.FC<SectionProps> = ({
 }) => {
   const [isCollapsed, setIsCollapsed] = useState<boolean>(!initiallyExpanded && true);
   return (
-    <StyledFormSection>
+    <StyledFormSection className="form-section">
```

Comment:
added this "marker" class to facilitate style overrides - For example this component defines a lot of padding and margin. Perhaps that kind of spacing should be left out to the parent containers - maybe discuss on Friday

Reply:
Yeah, I agree that the parent is the one that should figure out if the child needs spacing or not since it might be composed of multiple children,